app/src/main/java/org/simple/clinic/patient/PatientRepository.kt

```diff
@@ -95,6 +118,12 @@ class PatientRepository @Inject constructor(
 
   private fun savePatient(patient: Patient): Completable {
     return Completable.fromAction { database.patientDao().save(patient) }
+        .andThen(database.fuzzyPatientSearchDao().updateFuzzySearchTableForPatients(listOf(patient.uuid)))
+  }
+
+  private fun savePatients(patients: List<Patient>): Completable {
+    return Completable.fromAction { database.patientDao().save(patients) }
+        .andThen(database.fuzzyPatientSearchDao().updateFuzzySearchTableForPatients(patients.map { it.uuid }.distinct()))
```

Comment:
Hrm, why do we need `distinct` here? Don't we always get distinct patients for saving them?

Reply:
Right, good point. I don't know why I added that; maybe it was because I adding `distinct` clauses when merging the patient search results. I'll remove this.